containers/clojure/README.md

```diff
@@ -0,0 +1,134 @@
+# Clojure
+
+## Summary
+
+*Develop Clojure applications. Includes the Calva and clj-kondo extensions.*
+
+| Metadata                    | Value                 |
+|-----------------------------|-----------------------|
+| *Categories*                | Core, Languages       |
+| *Definition Type*           | Dockerfile            |
+| *Works in Codespaces*       | Yes                   |
+| *Container host OS support* | Linux, MacOS, Windows |
+| *Container OS*              | Debian                |
+| *Languages, platforms*      | Clojure               |
+
+## Using this definition
+
+While this definition should work unmodified, you can select the version of Java the container uses by updating the `VARIANT` arg in the included `devcontainer.json` (and rebuilding if you've already created the container).
+
+```json
+// Or you can use 16-bullseye or 16-buster if you want to pin to an OS version
```

Comment:
Here again I'm guessing this should be updated to Java 17?

Reply:
Updated!